services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/DynamoDbRetryPolicy.java

```diff
@@ -37,6 +38,12 @@
      **/
     private static final int LEGACY_MAX_ERROR_RETRY = 8;
 
+
+    /**
+     * Default max retry count for DynamoDB client, when using the STANDARD retry mode.
+     **/
+    private static final int STANDARD_MAX_ERROR_RETRY = 10;
```

Comment:
The legacy max error retry is 8 in v2, so the standard mode should be the same. We can probably just reuse the existing `LEGACY_MAX_ERROR_RETRY`

Reply:
Updated STANDARD_MAX_ERROR_RETRY as 8 , I would like to keep a separate Constant for Standard mode to avoid confusions

Reply:
Can we make it `private static final int STANDARD_MAX_ERROR_RETRY = LEGACY_MAX_ERROR_RETRY`? My concern is that if we ever change the legacy max error retry (although not likely), we might forget to change both places.

Reply:
Done